src/Features/Traits/Eloquent.php

```diff
@@ -4,23 +4,146 @@
 
 namespace Blumilk\BLT\Features\Traits;
 
+use Behat\Gherkin\Node\TableNode;
+use Blumilk\BLT\Helpers\LaravelRelations;
 use Illuminate\Support\Str;
 use PHPUnit\Framework\Assert;
 
 trait Eloquent
 {
     /**
-     * @Given there is :count :model object in database
-     * @Given there should be :count :model object in database
-     * @Given there are :count :model objects in database
-     * @Given there should be :count :model objects in database
+     * @Given there is a model :model in the database
```

Comment:
All method using tables should have `:` at the end of sentence. Here it's optional, so I would add two sentences.